packages/core/src/logger/defaultLogger.ts

```diff
@@ -0,0 +1,25 @@
+import { Level, LoggerInterface } from '@apimatic/core-interfaces';
+
+export class ConsoleLogger implements LoggerInterface {
+  public log(level: Level, message: string, params: Record<string, any>): void {
+    const formattedMessage = formatMessage(message, params);
+    // tslint:disable-next-line:no-console
+    console.log(level + ': ' + formattedMessage);
+  }
+}
+
+const formatMessage = (msg: string, obj: Record<string, any>): string => {
```

Comment:
Methods must be defined using the `function formatMessage(...)` syntax.

Reply:
ok